JavaArchitecturesAndDesignPatterns/patterns/behavioral/src/main/java/command/AccountingReport.java

```diff
@@ -0,0 +1,13 @@
+package command;
+
+public class AccountingReport extends Report {
+
+    public AccountingReport(Reports reports) {
+        super(reports);
+    }
+
+    @Override
+    public void Generate() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.